src/ui/Quickview/Quickview.ts

```diff
@@ -257,13 +257,13 @@ export class Quickview extends Component {
   private buildContent() {
     const icon = this.buildIcon();
     const caption = this.buildCaption();
-    this.buildTooltipIfNotInCardLayout(icon, caption);
-
     const content = $$('div');
+
     content.append(icon);
     content.append(caption);
-
     $$(this.element).append(content.el);
+
+    this.buildTooltipIfNotInCardLayout(icon, caption);
```

Comment:
What do you think about adding a test to check that the tooltip is not added when in card mode?

Reply:
Done